app/models/concerns/foreman_wreckingball/host_status_extensions.rb

```diff
@@ -7,7 +7,7 @@ def wreckingball_statuses
     end
 
     def find_wreckingball_status_by_host_association(host_association)
-      wreckingball_statuses.find { |s| s.host_association == host_association }
+      wreckingball_statuses.find { |s| s.host_association.to_s == host_association.to_s }
```

Comment:
Can we do this via active record, e.g. via a custom scope?

Reply:
I don't think we can use custom scope here. We have to go through statuses registry and find wreckingball status that match `host_association`.